generator/processor.go

```diff
@@ -459,14 +459,29 @@ func joinDirectory(directory string, files []string) []string {
 	return result
 }
 
-var goPath = os.Getenv("GOPATH")
+var goPaths = filepath.SplitList(build.Default.GOPATH)
+
+func getPkgSrcDir(importPath string) (string, error) {
+	wd, err := os.Getwd()
```

Comment:
I'm not really sure that the current working directory is the good thing to pass to build.Import, but it seems to do the job